src/grav_rfi_ompcuda.h

```diff
@@ -1,6 +1,14 @@
 #pragma once
 
-extern "C"  __declspec(dllexport) double* foo(int h_point_count, int h_prism_count, int h_lx, int h_ly, int h_lz, int h_kmax, int a, int b, double h_z_obs, double h_dz, double h_zmax,
+#ifdef _MSC_VER
+	#define DLLEXPORT __declspec(dllexport)
+#else
+	#define DLLEXPORT
+#endif
+
+extern "C" {
+	DLLEXPORT double* foo(int h_point_count, int h_prism_count, int h_lx, int h_ly, int h_lz, int h_kmax, int a, int b, double h_z_obs, double h_dz, double h_zmax,

```

Comment:
The revised could not be built successfully, so we edited here, putting "DLLEXPORT" first.

Reply:
lgtm